src/images/icons/bullet.svg

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
+  <title>bullet</title>
+  <g fill="none" fill-rule="evenodd">
```

Comment:
can we remove this g entirely?

Reply:
@Malinicz ignore that comment, looks we need it

Reply:
@Malinicz alternatively you can remove group along with path. Circle alone is enough

Reply:
@coderitual I wanted to set fill as `currentColor`, but then I checked other icons and most of them have fill set to `#000`. However, I left below and it seems to work fine:

```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
  <title>bullet</title>
  <circle cx="256" cy="256" r="64" fill="currentColor" fill-rule="evenodd"/>
</svg>
```

Reply:
@Malinicz could you give me an example of web icon having fill set to #000? if there is any it's a bug